initfiles/examples/embed/couchbase-simple.ecl

```diff
@@ -0,0 +1,112 @@
+IMPORT couchbase;
+/******************************************************************************************
+   This set of example queries are based on the couchbase sample bucket 'travel-sample'
```

Comment:
This is a read-only sample based on couchbase sample data, I plan on providing another set of queries based on iot data